ckanext/dbquery/blueprints/dbquery.py

```diff
@@ -18,17 +18,46 @@ def index():
     result = None
     request = toolkit.request
 
+    # Process form submission
     if request.method == 'POST':
         form = request.form
         query = form.get('query')
         if query:
             data_dict = {'query': query}
             result = toolkit.get_action('query_database')(None, data_dict)
 
+    # Get recent queries for sidebar
+    queries = toolkit.get_action('dbquery_executed_list')({}, {'limit': 5})
```

Comment:
Remove `limit`, it's not implemented (it'll be a good idea for a future PR)

Reply:
https://github.com/unckan/ckanext-dbquery/pull/9/commits/c67940c80fd916db1b936439be2d8d222ae44896